crates/clvm-derive/src/from_clvm.rs

```diff
@@ -151,8 +151,8 @@ fn check_rest_value(crate_name: &Ident, repr: Repr) -> TokenStream {
     match repr {
         Repr::Atom | Repr::Transparent => unreachable!(),
         // We don't need to check the terminator of a solution.
```

Comment:
I think it would be good to update this comment, to explain that consensus rules don't require lists to be nil terminated, and that we just ignore any additional elements we don't expect.